build.sbt

```diff
@@ -0,0 +1,31 @@
+name := "sttp-openapi-generator-example"
+
+version := "0.1"
+
+scalaVersion := "2.13.2"
+
+val root = file(".")
+
+lazy val petstoreApi = project
+  .in(file("petstore-api"))
+  .settings(
+    openApiInputSpec := s"${baseDirectory.value.getPath}/petstore.yaml",
+    openApiGeneratorName := "scala-sttp",
+    openApiOutputDir := baseDirectory.value.name,
+    openApiIgnoreFileOverride := s"$root/openapi-ignore-file",
```

Comment:
Maybe we can obtain the `root` in a more sbt-specific way? Relying on `.` can be error-prone

Reply:
Also would be good to have some comments on why we need this setting